javaparser-core/src/main/java/com/github/javaparser/ast/nodeTypes/NodeWithSimpleName.java

```diff
@@ -45,4 +46,6 @@ default N setName(String name) {
     default String getNameAsString() {
         return getName().getIdentifier();
     }
+
+    default NameExpr getNameExpr() {return new NameExpr(getName()); }
```

Comment:
I would put the return on a separate line. Alternatively I would put a space before the return